.rubocop.yml

```diff
@@ -12,7 +12,7 @@ AllCops:
     - 'vendor/**/*'
     - 'Dangerfile'
   ExtraDetails: true
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 2.6
```

Comment:
Should this be 2.7? When we went to 2.5 we had it at 2.5
 https://github.com/ualbertalib/pushmi_pullyu/commit/99b1d8f8882cdfcef153bbd19a5230b5ec969a4d

Reply:
Funny story.  I tried to make this 2.7 thinking that linting would be independent, but one of the nags was that it should match the required_ruby_version in pushmi_pullyu.gemspec.